bridge.go

```diff
@@ -1,8 +1,15 @@
-// Copyright (c) 2013, Prometheus Team
-// All rights reserved.
+// Copyright 2013 Prometheus Team
```

Comment:
Ah. Right. Then this needs to be fixed in the PGW as well, where I stole this header snippet from :)